### DISCUSSION TITLE
Add GraphQL Mutations support

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3052] [Feature Requests]

**Comment by hermesalvesbr** [comment]
This is very important feature.

**Reply by rijkvanzanten** [reply]
Then you should upvote it 😉 

Also, feel free to open a PR if you wanna take a swing at it 👍🏻

**Comment by rijkvanzanten** [comment]
This will be implemented and is tracked in https://github.com/directus/directus/issues/4561

**Comment by rijkvanzanten** [comment]
This has since been implemented!
